frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -210,11 +210,13 @@ const getOperandStatus = (obj: K8sResourceKind): OperandStatusType => {
     };
   }
 
-  const trueCondition = conditions?.find((c: K8sResourceCondition) => c.status === 'True');
-  if (trueCondition) {
+  const trueConditions = conditions?.filter((c: K8sResourceCondition) => c.status === 'True');
+  if (trueConditions?.length) {
+    const types = [];
+    trueConditions.map((c: K8sResourceCondition) => types.push(c.type));
```

Comment:
You need to assign the result

```suggestion
    const types = trueConditions.map((c: K8sResourceCondition) => types.push(c.type));
```